python/instrumentation/openinference-instrumentation-mistralai/pyproject.toml

```diff
@@ -37,7 +37,7 @@ instruments = [
   "mistralai",
 ]
 test = [
-  "mistralai",
+  "mistralai<0.2",
```

Comment:
what's the broken change? do we need a follow-up ticket?

Reply:
yes i filed ticket https://github.com/Arize-ai/openinference/issues/502